### PR TITLE
fix: #286 fix invoice boundaries for termination case

### PR DESCRIPTION
**Pull request by lovrocolic**

### CODE REVIEW

File: app/services/invoices/create_service.rb [hunk 1]
```diff
@@ -59,6 +59,19 @@ def from_date
...
                    else
                      raise NotImplementedError
+                   end
+
+      # NOTE: In case we are terminating old plan (paying in arrear),
+      # we should move to the beginning of the billing period
+      if subscription.terminated?
+        @from_date = case subscription.plan.interval.to_sym
+                     when :monthly
+                       Time.zone.at(timestamp).to_date.beginning_of_month
```
Comment (lovrocolic) [inline app/services/invoices/create_service.rb:69]: we already covered the case If it's less than started_at (below)

Comment (vincent-pochet) [reply app/services/invoices/create_service.rb:69]: What do you think of moving this logic into a dedicated method?
We start to have a lot of logic in this one

**Review by vincent-pochet — APPROVED** [review]
Looks good! :ship: it!

File: app/services/invoices/create_service.rb (hunk 1 above)
Comment (lovrocolic) [reply app/services/invoices/create_service.rb]: agree!

**Review by jdenquin — APPROVED** [review]

**Pull request closed without merging**